.goreleaser.yml

```diff
@@ -28,7 +28,9 @@ archives:
     builds: [macos, linux]
     <<: &archive_defaults
       name_template: "{{ .ProjectName }}_v{{ .Version }}_{{ .Os }}-{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
-    wrap_in_directory: true
+      files:
+        - none*
```

Comment:
lol I just read [the docs](https://goreleaser.com/archive/#packaging-only-the-binaries) about this... at least they are acknowledging that it's a hack.